rccheck/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+name = "rccheck"
+version = "0.1.0"
+license = "Apache-2.0"
+authors = ["Francois Garillot <francois@mystenlabs.com>"]
+description = "a custom verifier for pre-shared keys"
```

Comment:
Add a hint this concerns quic/tls? Second, the thing that delayed me understanding this is: as far I understand the pre-shared keys are PUBLIC KEYS, so the client knows the public key of the server, and the server know the public key of the client (initiator). NOT pre-shared secret keys, as the ones used in session resumption in TLS <=1.1. I think this is worth pointing out here.